packages/api/test/e2e/send.e2e.ts

```diff
@@ -1,17 +1,23 @@
-import { WsProvider } from "@polkadot/rpc-provider";
+import WebSocket from '@polkadot/x-ws';
+import { RpcCoder } from "@polkadot/rpc-provider/coder";
 
 describe('send', (): void => {
-    let ws: WsProvider;
 
-    beforeEach((): void => {
+    it('Send request using websocket', async (done)  => {
         const providerUrl = 'wss://cennznet.unfrastructure.io/public/ws';
-        ws = new WsProvider(providerUrl);
-    });
-
-    it('Send request using ws provider', async ()  => {
-        const hash = await ws.send('chain_getBlockHash', []);
-        expect(hash).toBeDefined();
-        const chain = await ws.send('system_chain', []);
-        expect(chain).toBe('CENNZnet Azalea');
+        const ws = new WebSocket(providerUrl);
+        const coder = new RpcCoder();
+        ws.onopen = function() {
+            const requestObject = coder.encodeJson('system_chain', []);
+            ws.send(requestObject);
+            ws.onmessage = function (event) {
+                expect(event.data).toEqual('{"jsonrpc":"2.0","result":"CENNZnet Azalea","id":1}');
```

Comment:
Nice, just we need to remember to update this soon as we are going back to hash ids instead of numbers. But hopefully, we keep that for long.